### PR TITLE
adds help link to navbar

### PR DESCRIPTION
**Pull request by LizabethPetersen**

### CODE REVIEW

File: src/components/navbar/navbar.js [hunk 1]
```diff
@@ -103,6 +103,9 @@ class Navbar extends React.Component {
                 <a className="dropdown-item" onClick={ this.props.doLogout }>Logout</a>
               </div>
             </li>
+            <li className="nav-item">
+              <a className="nav-link help" href="https://docs.google.com/presentation/d/e/2PACX-1vTRSMVBEvObOl1sCKPmXMChP8A4eZScVmrRrzS6mDw0Imi5LkbFd1sSgqDS-QEPcBD-gvBFwmanrPIC/pub?start=true&loop=false&delayms=3000"alt="Help Documentation" target="_blank" rel="noopener noreferrer">Help</a>
```
Comment (TCW417) [inline src/components/navbar/navbar.js]: Is this a permalink? or will it change/go obsolete when the document is updated?  I suggest testing this or resolving it before we add this link.

Comment (LizabethPetersen) [reply src/components/navbar/navbar.js]: I believe the document automatically updates; the Google slide deck is under continual "publish" mode. We can wait, though.

**Comment by TCW417** [comment]
Easy to test. Just edit the doc then follow the link to see what f you can see the edits.

File: src/components/navbar/navbar.js (hunk 1 above)
Comment (karenbtlai) [reply src/components/navbar/navbar.js]: can confirm, the link does not change even when document edit (often use it for school). But link may change when duplicated, or when sharing status is changed.

**Pull request merged**